prov/efa/src/efa_hmem.c

```diff
@@ -109,30 +212,23 @@ static int efa_domain_hmem_info_init_cuda(struct efa_domain *efa_domain)
 {
 #if HAVE_CUDA
```

Comment:
Do you still need HAVE_CUDA ? Same for HAVE_NEURON. I don't see it uses any cuda API symbols anymore.

Reply:
Good callout. We can get rid of the guards after this change. Please see update.